adapter/src/adapter.rs

```diff
@@ -59,19 +129,23 @@ impl FlinkODFAdapter {
             .args(
                 if let Some(prev_checkpoint_dir) = &request.prev_checkpoint_path {
                     vec![
-                        "run".to_owned(),
-                        "-s".to_owned(),
+                        "run".to_string(),
```

Comment:
At the first look, `to_owned()` & `to_string() do the same job, so I'd like to ask: 

**Q:** that is a stylistic change (easy to read) or do we have some more technical motivation to do that?

Reply:
AFAIK they do the same thing. When I just started with Rust I was using `to_owned()` but later heard that `to_string()` is more explicit and easier to search for, e.g. for identifying places where small string optimization or using `Write` would be beneficial to avoid extra allocations.

I don't have a strong enough preference to tell anyone to use one or another :)